lambdas/abandon/src/test/java/uk/gov/di/ipv/cri/kbv/api/handler/AbandonKbvHandlerTest.java

```diff
@@ -35,15 +39,19 @@
 class AbandonKbvHandlerTest {
     private static final String ABANDON_STATUS = "Abandoned";
     private static final String HEADER_SESSION_ID = "session-id";
-    private static final String ABANDON_KBV = "abandon_kbv";
+    private static final String ABANDON_KBV = "ABANDONED";
```

Comment:
Let keep this the way it was formerly, don't change the value from `abandon_kbv` to `ABANDONED`